### ISSUE TITLE
Move "get started" instructions to the bottom after installation

### ISSUE THREAD

**Issue by peterp** [redwoodjs/redwood#2128]
![image](https://user-images.githubusercontent.com/44849/112717219-d368f400-8ef3-11eb-8e21-edb9855fc91d.png)

I think it might be easier to parse if the most important information (the text I highlighted) is at the bottom of the installation instructions.

**Comment by semagraph** [comment]
Hey @peterp, do you want the 'Enter the directory...' line to go underneath the 'Find a Good First Issue' line?

**Comment by peterp** [comment]
I think maybe those two sections should flip around:

```
Thanks for trying Redwood

We've installed it here...

Type cd `path/to/redwood` and `yarn rw dev` to get started
```

**Comment by semagraph** [comment]
I was about to do a pull request but it looks like the change has already been made, guess I need to be a little quicker next time!

**Comment by Ayushsunny** [comment]
Hello @peterp I would like to work on this issue

**Comment by thedavidprice** [comment]
Hi all, apologies for the delayed reply. This change has not been made. Here is the code in `main` branch:
https://github.com/redwoodjs/redwood/blob/7be25811c04ce92a22863d841c05cf70d12682a3/packages/create-redwood-app/src/create-redwood-app.js#L170-L219 

If you would like to work on it, please reply in a comment and then do initiate a PR including `closes #2128` in the description.

Thanks in advance!

**Comment by qooqu** [comment]
I took a crack at it. PR here:

https://github.com/redwoodjs/redwood/pull/2278
